lib/API.js

```diff
@@ -674,11 +677,28 @@ class API {
 
     app_conf = appConf[0];
 
+
+    if (opts.ignoreWatch) {
+      flagWatch.handleFolders(opts.ignoreWatch, ignoreFileArray);
+      if (app_conf.ignore_watch) {
+        app_conf.ignore_watch = ignoreFileArray;
```

Comment:
Why is this overriding the entire `ignore_watch` flag? This effectively broke `--ignore-watch` in the CLI -- it now only accepts a single folder.

Reply:
Fix: https://github.com/Unitech/pm2/pull/4224